typegate/deno_import.json

```diff
@@ -20,6 +20,7 @@
     "oauth2_client": "https://deno.land/x/oauth2_client@v0.2.1/mod.ts",
     "test/mock_fetch": "https://deno.land/x/mock_fetch@0.3.0/mod.ts",
     "json_schema_typed": "https://deno.land/x/json_schema_typed@v8.0.0/draft_latest.ts",
-    "url_join": "https://deno.land/x/url_join@1.0.0/mod.ts"
+    "url_join": "https://deno.land/x/url_join@1.0.0/mod.ts",
+    "collections": "https://deno.land/std@0.139.0/collections/mod.ts"
```

Comment:
We already have `"std/"` on line 3; or do we need features from that specific (old) version?

Reply:
Imports could be:
```ts
import { mapValues } from "std/collections/map_values.ts";
```
This uses the version specified on the `"std/"` field, and download less files: only `map_values.ts` and its deps; `"collections/mod.ts"` would download all the imports in `mod.ts`.